packages/teams-js/src/internal/communication.ts

```diff
@@ -87,7 +87,8 @@ export function initializeCommunication(
   apiVersionTag: string,
 ): Promise<InitializeResponse> {
   // Listen for messages post to our window
-  CommunicationPrivate.messageListener = async (evt: DOMMessageEvent): Promise<void> => await processMessage(evt);
+  CommunicationPrivate.messageListener = async (evt: DOMMessageEvent): Promise<void> =>
+    await processIncomingMessage(evt);
```

Comment:
Should this be in curly braces now that it's on a new line?

Reply:
This was an automatic change that was done by prettier after the mechanical refactor that was the rename of `processMessage` but I can add the curlies now.

Reply:
After much discussion I actually removed the `async` and `await` since they are not necessary. `processMessage` returns a `Promise<void>` and so now it just keeps returning that up through the caller, enabling the caller to `await` it at their leisure, rather than enforcing the `await` to happen right here in this function.